### ISSUE TITLE
Feature Request: ability to modify the volume of the suggested drinks

### ISSUE THREAD

**Issue by mwip** [z3r0c00l-2k/AquaDroid#29]
usually I have a large cup of tea which contains about 320 ml. I think it would be a nice feature to be able to modify the volume of the suggested drinks (or even add your own custom ones). modifying could be enabled by long-tap or alike.

**Comment by z3r0c00l-2k** [comment]
I have added option for custom volume in this version [v1.3](https://github.com/z3r0c00l-2k/AquaDroid/releases/tag/v1.3). Please check it

**Comment by mwip** [comment]
Awesome, thanks! Have you pushed it to f-droid yet?

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by z3r0c00l-2k** [comment]
> Awesome, thanks! Have you pushed it to f-droid yet?

something is wrong with my f-droid config. f-droid detecting my new version

**Comment by SebiderSushi** [comment]
The new feature is very handy.

However, i think it would still be useful to implement editing the amount on regular tiles alongside this new feature.

This would fully support the two use cases:
- Adding a variable, spontaneous amount once in a while
- Adding the custom, fixed volume of your favourite cup or bottle multiple times a day

**Comment by add1989** [comment]
I agree. For example, a typical canned soft drink in the UK is 330ml and I would like to be able to modify the 200ml icon to the correct volume.

**Comment by z3r0c00l-2k** [comment]
Will do it

**Comment by LongJohn-Silver** [comment]
Any updates?

**Comment by manto89** [comment]
Hi,
I'm not an experience developer but i've worked on this issue and came up with the idea of long-pressing on the suggested values to change them. The edits are saved in SharedPreferences.
I'm pushing the commit on my fork and will make a PR with this and other issues i'm working on.
Let me know if you have any comments/issues

**Comment by gary-host-laptop** [comment]
Are you planning on reviving the app, @manto89? If that is the case maybe it would be better to keep the discussions over there.

**Comment by manto89** [comment]
@LongJohn-Silver It's something I'm considering although I'm not very experienced, particularly on the release process.
Let's see if @z3r0c00l-2k answers in the next few days, otherwise I'll arrange something to move the development to my repo.

**Comment by z3r0c00l-2k** [comment]
I am sorry guys, I got very busy in the past year. @manto89 feel free to fork this. And Good luck 👍

**Comment by manto89** [comment]
Hi @z3r0c00l-2k I've sent u an email to find the best way to migrate everything (issues, fdroid release, etc) in order to not bother you with PR and merges.
Let me know what do you think about it
